README.md

```diff
@@ -1,2 +1,15 @@
 # Bookmark-Telegram-Bot
 A telegram bot to organize and save useful links, mainly for twitter bookmark
+
+# Requirement
+- Node.js "https://nodejs.org/en/download"
+- grammY "https://grammy.dev"
```

Comment:
remove this requirement (grammY) since the only thing user should care is having nodejs installed, the library is automatically handled by npm.